vulnerable_people_form/form_pages/shared/answers_enums.py

```diff
@@ -22,8 +22,8 @@ class ViewOrSetupAnswers(EnumWithText):
 
 @enum.unique
 class ApplyingOnOwnBehalfAnswers(EnumWithText):
-    YES = 1
-    NO = 0
+    YES = 0
```

Comment:
is there any reason these have been swapped around?

Reply:
Ignore this comment, i have now seen the slack conversation confirming that the underlying enum value needs to be sapped arround